### ISSUE TITLE
plugin installation fails (Can't find Python executable)

### ISSUE THREAD

**Issue by germsb** [universal-vue/uvue#1]
**Describe the bug**
plugin installation fails (vue-cli ui) 

```
+ @uvue/vue-cli-plugin-ssr@0.1.0-alpha.1
added 37 packages from 89 contributors and audited 28871 packages in 11.594s
found 0 vulnerabilities


🚀  Invoking generator for @uvue/vue-cli-plugin-ssr...
📦  Installing additional dependencies...


> node-zopfli-es@0.0.2 install C:\dev\uvue-msp\node_modules\node-zopfli-es
> node-gyp rebuild


C:\dev\uvue-msp\node_modules\node-zopfli-es>if not defined npm_config_node_gyp (node "C:\Users\germs\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Users\germs\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (C:\Users\germs\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\configure.js:484:19)
gyp ERR! stack     at PythonFinder.<anonymous> (C:\Users\germs\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\configure.js:509:16)
gyp ERR! stack     at C:\Users\germs\AppData\Roaming\npm\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:158:21)
gyp ERR! System Windows_NT 10.0.17134
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\germs\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\dev\uvue-msp\node_modules\node-zopfli-es
gyp ERR! node -v v10.5.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-zopfli-es@0.0.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-zopfli-es@0.0.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\germs\AppData\Roaming\npm-cache\_logs\2018-11-10T12_50_44_592Z-debug.log
 ERROR  command failed: npm install --loglevel error
```
Please indicate versions of:
```
node: 10.5.0
vue-cli: 3.1.1
uvue: 0.1.0-alpha.1
os: win 10

```

**Comment by chymz** [comment]
Oh yeah ! forgot to remove `shrink-ray-current` package.

It will be fixed in next update

**Comment by chymz** [comment]
Fixed in `0.1.0-alpha.2`
